reflex/components/base/body.pyi

```diff
@@ -1,21 +1,85 @@
-"""Stub file for body.py"""
+"""Stub file for reflex/components/base/body.py"""
 # ------------------- DO NOT EDIT ----------------------
 # This file was generated by `scripts/pyi_generator.py`!
 # ------------------------------------------------------
 
-from typing import Optional, Union, overload
-from reflex.components.component import Component
+from typing import Any, Literal, overload, Dict, Optional, Union, List
 from reflex.vars import Var, BaseVar, ComputedVar
-from reflex.event import EventHandler, EventChain, EventSpec
+from reflex.event import EventChain, EventHandler, EventSpec
+from reflex.style import Style
+from reflex.components.component import Component
 
 class Body(Component):
+    ...
```

Comment:
oops, these shouldn't be here if the class is not empty...we should move these lines (https://github.com/reflex-dev/reflex/blob/9e5a47b52d51b7a60608355515de82a57ee2ba69/scripts/pyi_generator.py#L449-L451) to the end of that `visit_ClassDef` function to ensure we dont add the ellipsis when we will generate a function.